tools/checkTheme.js

```diff
@@ -184,6 +191,79 @@ function check_group(group, rules) {
   }
 }
 
+const round2 = (x) => Math.round(x*100)/100;
+
+class CSSRule {
+  constructor(rule, body) {
+    this.rule = rule;
+    if (rule.declarations) {
+      this.bg = rule.declarations.find(x => x.property =="background")?.value;
+      this.fg = rule.declarations.find(x => x.property =="color")?.value;
```

Comment:
Two notes on this one:

1. Some styles use `background-color` to differentiate the background for specific elements. I think we should add support for checking `background-color` for WCAG contrast.
	https://github.com/highlightjs/highlight.js/blob/0bf3adeecd5ba76cef8ee843e38722089b232da1/src/styles/foundation.css#L74-L77

2. Should we have a minimum Node defined somewhere for dev environments? Optional chaining was added in Node 14 and our `package.json` requires Node 12+ (correctly so, for our users).

Reply:
> Should we have a minimum Node defined somewhere for dev environments? 

Ah, we probably should, but where... damn old versions of Node. :-)

> Some styles use background-color  

I'll break this out into a separate issue... the tool doesn't have to be perfect for the main thrust of this to be mergeable.